styles/style.css

```diff
@@ -0,0 +1,699 @@
+  text-transform: uppercase;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding-bottom: 10px;
+}
+
+.star-item span {
+  font-size: 10px;
+  line-height: 12px;
+  opacity: .6;
+  padding-top: 5px;
+}
+
+.star-group {
+  width: 230px;
+  padding-left: 30px;
+}
+
+.rating_icon {
+  width: 280px;
+  height: 90px;
+  margin-bottom: 50px;
+}
+
+.safety_info-btn {
+  width: 319px;
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+  white-space: nowrap;
+}
+
+.safety_info-text {
+  font-size: 13px;
+  font-weight: 600;
+  font-style: normal;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.safety-info__img {
+  margin-right: 5px;
+}
+
+.btn-safety {
+  font-size: 13px;
+  font-weight: bold;
+  font-style: normal;
+  text-transform: uppercase;
+  color: #fff;
+  display: block;
+  margin-left: 10px;
+  padding: 10px 43px; 
+  border: 2px solid #fff;
+  background: #E51937;
+  cursor: pointer;
+  transition-duration: .4s;
+}
+
+.btn-safety:hover {
+  background: #fff;
+  color: #E51937;
+}
+
+.safety_car-frame {
+  width: 100%;
+  position: relative;
+  top: 14vw;
+  right: 7vw;
+}
+
+.safety-img {
+  background-image: url(../img/Model_S-1.png);
+  background-repeat: no-repeat;
+  background-position-x: 104%;
+  background-position-y: 25%;
+  background-size: 85%;
+}
+
+.whs-img{
+  width: 94px;
+  position: relative;
+  top: 17vw;
+  left: 58vw;
+}
+
+@media screen and (max-width: 1080px) {
+
+  .header-models {
+    font-size: 12px;
+  }
+
+  .header-menu {
+    font-size: 12px;
+  }
+
+  .car-tesla {
+    width: 73%;
+    top: 0;
+    right: 0;
+  }
+
+  .first-floor__item {
+    background-position-y: 15%;
+  }
+
+  .advantage-info {
+    max-width: 1200px;
+    margin: 0 auto;
+    background-position: top left;
+  }
+
+  .top-title {
+    width: 100%;
+    text-align: right;
+    margin-right: 50px;
+  }
+
+  .info__description-advantage {
+    display: flex;
+    justify-content: space-between;
+    flex-wrap: nowrap;
+    max-width: 100%;
+    margin-top: 67%;
+  }
+
+  .whs {
+    display: none;
+  }
+}
+
+@media screen and (max-width: 950px) {
+
+  .nav {
+    display: flex;
+    justify-content: space-between;
+    padding: 15px 30px;
+  }
+
+  .header {
+    margin-bottom: 35px;
+  }
+
+  .header-models {
+    display: none;
+  }
+
+  .first-floor__item {
+    background-position-y: 15%;
+  }
+
+  .stat {
+    width: 100%;
+    margin: 0 auto 50px;
+  }
+
+  .params {
+    display: flex;
+    flex-flow: column;
+  }
+
+  .stat-power {
+    margin-right: 160px;
+  }
+
+  .params {
+    padding: 0 100px 0 150px;
+  }
+
+  .wrap-params-one {
+    margin-bottom: 36px;
+  }
+
+  .stat__test-drive {
+    width: 140px;
+  }
+
+  .test-drive {
+    display: none;
+  }
+
+  .test-drive__red {
+    display: block;
+  }
+
+  .btn__first-floor {
+    background-color: #E51937;
+    color: #fff;
+  }
+}
+
+@media screen and (max-width: 770px) {
+   .nav {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Privezentsev/pullRequest?prid=5491648)